org-static-blog.el

```diff
@@ -182,6 +182,16 @@ See also `org-static-blog-preview-ellipsis' and
   :type '(boolean)
   :safe t)
 
+(defcustom org-static-blog-preview-start nil
+  "Marker indicating the beginning of a post's preview."
```

Comment:
This should probably include a note about the meaning of `nil`, much like in `org-static-blog-rss-max-entries`. I'm assuming if `nil`, the "legacy" behavior of the first paragraph is used?